pkg/logs/sender/config.go

```diff
@@ -24,6 +24,10 @@ var logsEndpoints = map[string]int{
 	"agent-intake.logs.datad0g.eu":    443,
 }
 
+func IsSetAndNotEmpty(config config.Config, key string) bool {
```

Comment:
this method could be private, here you exposed it at package level.

Reply:
TIL about go encapsulation. Thanks for the comment, updated.